tsdb/index/index_test.go

```diff
@@ -553,3 +553,14 @@ func TestSymbols(t *testing.T) {
 	}
 	testutil.Ok(t, iter.Err())
 }
+
+func TestLabelNames(t *testing.T) {
```

Comment:
We are already testing label names in `TestDB_LabelNames`. This is not required.